thunder.profile

```diff
@@ -5,9 +5,22 @@
  * Enables modules and site configuration for a thunder site installation.
  */
 
+use Drupal\Core\Extension\Extension;
 use Drupal\Core\Form\FormStateInterface;
 use Drupal\block\Entity\Block;
 
+/**
+ * Implements hook_system_info_alter().
+ */
+function thunder_system_info_alter(array &$info, Extension $file, $type) {
+  // Thunder can not work properly without these modules. So they are enforced
+  // to be enabled.
+  $required_modules = ['config_selector'];
```

Comment:
I've ensure that if the config_selector is not installed for any reason this breaks nothing. Once the module is installed this ensures that it can not be uninstalled. Either by the UI or Drush.